src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -82,18 +91,19 @@ export class Artwork extends React.Component<Props, State> {
 
   componentDidMount() {
     this.markArtworkAsRecentlyViewed()
+    this.loadFullArtwork()
   }
 
   componentDidUpdate(prevProps) {
     // If we are visible, but weren't, then we are re-appearing (not called on first render).
     if (this.props.isVisible && !prevProps.isVisible) {
-      this.forceRefetch()
+      this.loadFullArtwork()
```

Comment:
I think I understand this change – it's because we don't expect above-the-fold content to need to be refreshed when the view re-appears? Because it should be static (unlike below-the-fold content, which could change, like current bid or sale status). We should add a comment here about that assumption 👍

Reply:
One of the above-the-fold components does need fresh data (`CommercialInformation`) but we do a couple of things to make sure it does indeed get the fresh data:

1. the 'full' query contains all the fields for both above and below the fold. This should update the relay cache entry for the artwork, which in turn should update the `artworkAboveTheFold` prop.
2. just to make doubly-sure, we pass the 'fullArtwork' value into the `CommercialInformation` component when it is not null.

I agree this is a little non-obvious so I'll add some comments and a test :+1:

Reply:
Added a test for the cache behaviour (1), and it works so I removed (2) and also simplified the full artwork query by reusing the `Artwork_artworkAboveTheFold` fragment 🚀